_resources/port1.0/group/linear_algebra-1.0.tcl

```diff
@@ -94,9 +96,20 @@ variant atlas conflicts accelerate blis flexiblas openblas description {Build wi
 
 variant blis conflicts accelerate atlas flexiblas openblas description {Build with linear algebra from BLIS} {
     depends_lib-append      port:blis
-    linalglib               -lblis
-    cmake_linalglib         -DBLAS_LIBRARIES=blis \
-                            -DLAPACK_LIBRARIES=blis
+    if {!$blas_only} {
+        depends_lib-append  port:libflame
+        linalglib           -lblis -lflame
+        cmake_linalglib     -DBLAS_LIBRARIES=blis \
+                            -DLAPACK_LIBRARIES=flame
+        # Notice, that some ports may link against blis + lapack instead.
+        # depends_lib-append  port:lapack
+        # linalglib           -lblis -llapack
+        # cmake_linalglib     -DBLAS_LIBRARIES=blis \
+        #                     -DLAPACK_LIBRARIES=lapack
+    } else {
+        linalglib           -lblis
+        cmake_linalglib     -DBLAS_LIBRARIES=blis
+    }
     # cmake_linalglib         -DBLA_VENDOR=FLAME
```

Comment:
please don't leave commented out code

Reply:
I dropped this line. (The chunk above is actually useful.)

Reply:
useful for what and/or whom?